### PR TITLE
[TECHNICAL SUPPORT] LPS-98609 Liferay Auto Fields does not work with localized inputs

### PR DESCRIPTION
**Pull request by NemethNorbert**
Hey,

[LPS-98609](https://issues.liferay.com/browse/LPS-98609),
I think we reached a solution with @antonio-ortega, could you review the changes?

Thanks,

/cc @antonio-ortega

### CODE REVIEW

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/auto_fields.js [hunk 1]
```diff
@@ -155,10 +147,31 @@ AUI.add(
 
 					var formValidator = instance._getFormValidator(node);
 
+					var paletteIsCloned =
+						clone
+							.getDOM()
+							.querySelector("[id$='PaletteBoundingBox']") !=
```
Comment (jbalsas) [inline modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/auto_fields.js:153]: Can't we do `.one` instead?

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/auto_fields.js [hunk 2]
```diff
@@ -155,10 +147,31 @@ AUI.add(
...
+						var trigger = clone.one('button');
+
+						var list = A.Node.create(
+							'<ul class="dropdown-menu dropdown-menu-left-side"></ul>'
+						);
+
+						trigger.placeAfter(list);
+
+						list.append(palette);
+					}
```
Comment (jbalsas) [inline modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/auto_fields.js:174]: Missing additional line

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/auto_fields.js (hunk 1 above)
Comment (NemethNorbert) [reply modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/auto_fields.js:153]: Absolutely, I rebase it and push it soon.

**Comment by jbalsas** [comment]
Hey @antonio-ortega, @NemethNorbert, without thoroughly testing this, changes look good to me. 

I have the feeling that the cloning process could be simplified and made a bit more robust... I'm not particularly happy with the explicit node creation, since that's implementation specific and might break anytime that specific markup changes. 

In any case, if you've tested this and are confident, **feel free to push it forward**. I just added a couple of minor comments, but other than that it should be good to go.

File: modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/auto_fields.js (hunk 2 above)
Comment (NemethNorbert) [reply modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/auto_fields.js:171]: fixed

**Comment by jbalsas** [comment]
ci:test:forward

**Comment by jbalsas** [comment]
> The test will not be initiated because ci:test:forward is not a valid command

Duh!

**Comment by jbalsas** [comment]
ci:forward

**Comment by NemethNorbert** [comment]
c:forward

**Comment by NemethNorbert** [comment]
ci:test:relevant

**Comment by jbalsas** [comment]
ci:test:relevant

**Comment by NemethNorbert** [comment]
ci:test:relevant

**Comment by jpince** [comment]
@david-gutierrez-mesa consistent failure in `ContentPageConverter#ConvertWidgetPageWithNonStandardLayout`, is there a quarantine in progress?

**Comment by david-gutierrez-mesa** [comment]
@jpince this test was ignored in https://issues.liferay.com/browse/LRQA-57536. It was failing due to https://issues.liferay.com/browse/LPS-111348

**Comment by NemethNorbert** [comment]
ci:test:relevant

**Comment by jbalsas** [comment]
ci:forward

**Pull request closed without merging**